specs/commandline/policy.md

```diff
@@ -0,0 +1,129 @@
+                "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Dev, CN=wabbit-networks.io"
+            ]
+        },
+        {
+            "name": "wabbit-networks-prod",                             // Name of the second policy statement.
+            "registryScopes": [ "prod.wabbitnetworks.io/net-monitor" ],       
+            "signatureVerification": {                                
+                "level": "permissive"
+            },
+            "trustStores": [ "ca:wabbit-networks-prod" ],                  
+            "trustedIdentities": [                                    
+                "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Prod, CN=wabbit-networks.io"
+            ]
+        }
+    ]
+}
+```
+
+The goal of `notation policy` command is to provide a user-friendly CLI for users to manage trust policies without the knowledge of trust policy configuration name, directory path and property names. Two high level use cases as follows:
+
+1. Users can import trust policies from a JSON file, and export trust policies into a JSON file.
+2. Users can add/view/update/delete trust policies and properties without editing the trust policy configuration file.
+
+A phased approach is adopted to achieve the goal. The first use case will be implemented in phase-1. A trust policy template file will be provided for users to get started. This specification only covers phase-1.
+
+## Outline
+
+### notation policy command
+
+```text
+Manage trust policies for signature verification.
+
+Usage:
+  notation policy [command]
+
+Available Commands:
+  export    export trust policies to a JSON file
+  import    import trust policies from a JSON file
+  show      show trust policies
+
+Flags:
+  -h, --help   help for policy
+```
+
+### notation policy export
+
+```text
+Export trust policies to a JSON file
+
+Usage:
+  notation policy export [flags] <file_path>
+
+Flags:
+  -h, --help    help for export
+```
+
+### notation policy import
+
+```text
+Import trust policies from a JSON file
+
+Usage:
+  notation policy import [flags] <file_path>
+
+Flags:
+  -h, --help    help for import
+```
+
+### notation policy show
+
+```text
+Show trust policies
+
+Usage:
+  notation policy [flags] show
+
+Flags:
+  -h, --help    help for show
+```
+
+## Usage
+
+### Import trust policies from a JSON file
+
+```shell  
+notation policy import ./my_policy.json
+```
+
+The trust policies in the JSON file will be validated according to [trust policy properties](https://github.com/notaryproject/notaryproject/blob/v1.0.0-rc.2/specs/trust-store-trust-policy.md#trust-policy-properties). Upon successful import, the trust policies are printed out.
+
+### Export trust policies into a JSON file
+
+```shell
+notation policy export ./policy_exported.json
+```
+
+For phase-1, to update trust policies, users need to export the trust policies to a file first, update the file, and import the file again.
```

Comment:
There is a usecase where users have stored trust policy in some other location and they just want to import it.

Also, as mentioned earlier we should remove the `phase-1` string because the user doesn't know which phase we are in and we might forget to remove phase-1 when we implement phase-2.